### ISSUE TITLE
http://websocketd.com does not resolve

### ISSUE THREAD

**Issue by slloyd88** [joewalnes/websocketd#313]
Your website that you have on github and in the application no longer works.  http://websocketd.com

I propose you just change the URL to your github page.

**Comment by asergeyev** [comment]
it works for me I'm not sure if you had it temporarily offline

**Comment by slloyd88** [comment]
yeah, it looks like Norton ConnectSafe DNS servers are blocking your website.
